aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -224,6 +225,21 @@ func flattenFirehoseElasticsearchConfiguration(description *firehose.Elasticsear
 	return []map[string]interface{}{m}
 }
 
+func flattenVpcConfiguration(description *firehose.VpcConfigurationDescription) []map[string]interface{} {
+	if description == nil {
+		return []map[string]interface{}{}
+	}
+
+	m := map[string]interface{}{
+		"vpc_id":             aws.StringValue(description.VpcId),
+		"subnet_ids":         flattenStringList(description.SubnetIds),
+		"security_group_ids": flattenStringList(description.SecurityGroupIds),
```

Comment:
Can use `flattenStringSet` here for consistency with the `expandStringSet` below.